.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+python:
+- "2.7"
```

Comment:
I think it's better to explain this `allow_failures`:
https://github.com/travis-ci/travis-ci/issues/5113

in stead of removing 2.7